build.gradle

```diff
@@ -9,38 +9,32 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'net.minecraftforge.gradle:ForgeGradle:2.0-SNAPSHOT'
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-SNAPSHOT'
     }
 }
 apply plugin: 'net.minecraftforge.gradle.forge'
-*/
-
+
+/*
 // for people who want stable
 plugins {
     id "net.minecraftforge.gradle.forge" version "2.0.2"
 }
-
-version = mcversion + '-' + version_major + '.' + version_minor + '.' + version_patch
+*/
+version = mcversion + '-' + version_major + '.' + version_minor + '.' + version_patch
 group= "hunternif.mc.atlas" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
 archivesBaseName = "antiqueatlas"
 
 minecraft {
-    version = mcversion + '-' + forgeversion

```

Comment:
I thought this was convention for when there are several mods in the workspace. Though I guess I  could do with a symlink to the global "run" dir.